### PR TITLE
Add Halloween 2022 Fanart Voting Open news post

### PR DESCRIPTION
**Pull request by Ephemeralis**
Late for a very important date.

### CODE REVIEW

File: news/2022-10-24-halloween-fanart-voting-open.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: "Halloween 2022 Fanart Contest Voting Open"
+date: 2022-10-24 06:00:00 +0000
+---
+
+True horror strikes again - **109** incredible entries, but only **15** votes each to go around!
```
Comment (Walavouchey) [inline news/2022-10-24-halloween-fanart-voting-open.md:7]:
```suggestion
True horror strikes again — **109** incredible entries, but only **15** votes each to go around!
```

File: news/2022-10-24-halloween-fanart-voting-open.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+![](https://assets.ppy.sh/contests/154/header.jpg)
+
+The hallowed eve of the year's spookiest month is almost upon us, and it is now time to shuffle forth into the voting phase and help decide which ones will make it to the main menu.
+
+Check out a quick preview video below (courtesy of one Mr. Fat) of what's up for the vote set to the tunes of [cYsmix](https://osu.ppy.sh/beatmaps/artists/2)'s Fright March, but beware, for you might encounter horrors beyond compare:
```
Comment (Walavouchey) [inline news/2022-10-24-halloween-fanart-voting-open.md:13]:
```suggestion
Check out a quick preview video below (courtesy of one Mr. Fat) of what's up for the vote set to the tunes of [cYsmix](https://osu.ppy.sh/beatmaps/artists/2)'s *Fright March*, but beware, for you might encounter horrors beyond compare:
```

File: news/2022-10-24-halloween-fanart-voting-open.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    <video width="90%" controls>
+        <source src="https://assets.ppy.sh/contests/154/voting_showcase.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+[**Click here to go to the voting for the Halloween 2022 Fanart Contest!**](https://osu.ppy.sh/community/contests/154)
+
+We're unexpectedly tight on time this year, so the voting will only stay open for **5 days** instead of the usual 7. Make sure to get in quick and choose your favourites!
+
+A selection from the top voted entries will make it to the game's main menu as the Halloween 2022 Seasonal backgrounds, and their creators will also receive 3 months of osu!supporter.
```
Comment (Walavouchey) [inline news/2022-10-24-halloween-fanart-voting-open.md:25]:
```suggestion
A selection from the top-voted entries will make it to the game's main menu as the Halloween 2022 Seasonal backgrounds, and their creators will also receive 3 months of osu!supporter. 
```

**Review by peppy — APPROVED** [review]

File: news/2022-10-24-halloween-fanart-voting-open.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: "Halloween 2022 Fanart Contest Voting Open"
+date: 2022-10-24 06:00:00 +0000
+---
+
+True horror strikes again — **109** incredible entries, but only **15** votes each to go around!
```
Comment (minusQuantumNeko) [inline news/2022-10-24-halloween-fanart-voting-open.md:7]: Entries number fix, there are only 108 entries on voting page
```suggestion
True horror strikes again — **108** incredible entries, but only **15** votes each to go around!
```

**Pull request merged**